server/controllers/authController.js

```diff
@@ -3,11 +3,11 @@ import moment from 'moment';
 import jwt from 'jsonwebtoken';
 import dotenv from 'dotenv';
 import bcrypt from 'bcryptjs';
-import User from '../models/User';
+import User from '../Models/User';
 import db from '../data/connection';
-import validate from '../helpers/validation';
+import validate from '../Helpers/Validation';
```

Comment:
Why are you importing the helpers with a capital letter at the start?

Reply:
I did that because I want to see them better. They contain almost all the back functionalities of the application, that's why I renamed all Controllers, Middleware and Helpers to have a capital letter at the start.